content/blog/2023-10-31.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "2023-10-31"
+date = 2023-10-31
++++
+
+---
+
+Today is Tuesday, day 304 of 365 (83.28%). We're in week 44 of 52 (84.61%). 
+
+## Planning
+
+What I am going to do today: 
+
+- [x] send a pull request for [flutter-examples#issue95](https://github.com/nisrulz/flutter-examples/issues/95)
+- [ ] send a pull request for [flutter-examples#issue22](https://github.com/nisrulz/flutter-examples/issues/22)
+
+## Activities executed
+
+Today I edited the README for the [calculator project](https://github.com/OmnicodeSolutions/flutter_codelab/tree/main/calculator) and sent the [PR for flutter-examples#issue95](https://github.com/nisrulz/flutter-examples/pull/143). I also worked on the PR for #issue22, which is the [ci_integration project](https://github.com/OmnicodeSolutions/flutter_codelab/tree/main/ci_integration), but I had some issues with the release process since it was completely new to me, so I wasn't able do send finish that part on the gh-actions, therefore I didn't send the PR.
+
+## Learnings
+
+Today I learned how to use gh-actions to lint and test a flutter project, I also studied a little bit about [geolocation plugin in Flutter](https://pub.dev/packages/geolocator) and deployment of both [iOS](https://docs.flutter.dev/deployment/ios) and [Android](https://docs.flutter.dev/deployment/android) apps.
```

Comment:
You also learned about

- https://semver.org/
- https://docs.python.org/release/3.11.6/whatsnew/changelog.html#python-3-11-6 (changelog)
- https://git-scm.com/book/en/v2/Git-Basics-Tagging git tags